corehq/apps/accounting/models.py

```diff
@@ -1524,6 +1528,20 @@ def get_total(self):
         return self.balance
 
 
+class WirePrepaymentInvoice(WireInvoice):
+    class Meta:
+        proxy = True
+
+    items = []
+
+    @property
+    def is_prepayment(self):
+        return True
+
+    def add_items(self, items):
+        self.items = items
```

Comment:
Is this a standard interface? It seems strange to call a setter (that *replaces* items) `add_items`. Why not just set the attribute directly at the place where this would be called (`payment.items = ...`) rather than using a setter?